server/src/main/java/org/opensearch/cluster/metadata/Metadata.java

```diff
@@ -180,11 +180,6 @@ static Custom fromXContent(XContentParser parser, String name) throws IOExceptio
             // handling any Exception is caller's responsibility
             return parser.namedObject(Custom.class, name, null);
         }
-
-        static Custom fromXContent(XContentParser parser) throws IOException {
```

Comment:
This looks to be marked as a `PublicApi` and would be breaking for `2.x`.
We might not use it but third party plugins and modules might be consuming it.

Reply:
@kotwanikunal it was added couple days back in https://github.com/opensearch-project/OpenSearch/pull/12190 . This PR is not even backed to 2.x